core/src/test/scala/unit/kafka/zk/ZKEphemeralTest.scala

```diff
@@ -47,7 +47,7 @@ object ZKEphemeralTest {
 
 @RunWith(value = classOf[Parameterized])
 class ZKEphemeralTest(val secure: Boolean) extends ZooKeeperTestHarness {
-  val jaasFile = kafka.utils.JaasTestUtils.writeZkFile()
+  val jaasFile = kafka.utils.JaasTestUtils.writeJaasContextsToFile(kafka.utils.JaasTestUtils.zkSections)
```

Comment:
I refactored the JaasTestUtils to get only one method which writes the JaasContext to file. So the kafka.utils.JaasTestUtils.zkSections creates the JaasSection for ZK but does not write it to file, like the writeZkFile did.